ingestor/service.go

```diff
@@ -372,6 +377,15 @@ func (s *Service) HandleTransfer(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
+	for _, prefix := range s.dropFilePrefixes {
+		if strings.HasPrefix(filename, prefix) {
+			logger.Errorf("Transfer request rejected due to prefix %q: %q", prefix, filename)
```

Comment:
I think this could get very noisy in the logs.

Reply:
Very true. Wanted some way to easily track this, but think tracing is probably a better mechanism in the future.